tests/integration/targets/aci_l3out_logical_node/tasks/main.yml

```diff
@@ -130,8 +152,8 @@
 
   # QUERY A SPECIFIC NODE
   - name: Query a specific node
-    cisco.aci.aci_l3out_logical_node:
-      <<: *node_update
+    aci_l3out_logical_node:
+      <<: *node_remove_loopback
```

Comment:
Should we assert also that query calls return the loopback child class and expected configuration?

Reply:
yes, you are right, we should assert this.